includes/06-ajax.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * WordPress ajax handlers
+ *
+ * @package ubc_wp_vote
+ */
+
+namespace UBC\CTLT\WPVote\Ajax;
+
+// If this file is called directly, abort.
+if ( ! defined( 'ABSPATH' ) ) {
+	die;
+}
+
+// Hooks.
+add_action( 'wp_ajax_ubc_ctlt_wp_vote_upvote', __NAMESPACE__ . '\\ubc_ctlt_wp_vote_upvote' );
+add_action( 'wp_ajax_ubc_ctlt_wp_vote_downvote', __NAMESPACE__ . '\\ubc_ctlt_wp_vote_downvote' );\
+add_action( 'wp_ajax_ubc_ctlt_wp_vote_rating', __NAMESPACE__ . '\\ubc_ctlt_wp_vote_rating' );
+
+/**
+ * Ajax handler for upvote toggle action.
+ *
+ * @return void
+ */
+function ubc_ctlt_wp_vote_upvote() {
+	check_ajax_referer( 'ubc', 'security' );
+
+	$object_id = isset( $_POST['object_id'] ) ? intval( $_POST['object_id'] ) : null;
+	$object_type = isset( $_POST['object_type'] ) ? sanitize_text_field( wp_unslash( $_POST['object_type'] ) ) : null;
```

Comment:
(Random aside: We should look at your WPCS settings, this should have thrown a warning that the `=` weren't aligned ;) )